### DISCUSSION TITLE
Cancel screen capture

### DISCUSSION THREAD

**Discussion by JeopardyTempest** [pubpub-zz/ppInk#137] [Q&A] [answered]
Suggest that pressing escape during the selection of a screen capture area cancel the capture, as this seems fairly regular UI?

**Comment by pubpub-zz** [comment, marked as answer]
Escape already stops process after you have entered screen capture. before and after starting the window. 

Also, remember that your drawings will be automatically saved in autosave.strokes.txt when exiting drawing, and you can reload them in the next session depressing the load button.

**Comment by JeopardyTempest** [comment]
Oye, that's on me, I had a hotkey active which was stealing the escape unnecessarily :-/  Sorry for the distraction, great work you're doing.

**Comment by pubpub-zz** [comment]
No Prob
Happy you are enjoying. do not hesitate to Star it 😊
